pkg/soc/soc_test.go

```diff
@@ -25,8 +26,8 @@ func TestToChunk(t *testing.T) {
 
 	id := make([]byte, 32)
 
-	payload := []byte("foo")
-	ch, err := chunk(payload)
+	payload := []byte("greaterthanspan")
```

Comment:
This actually does not need to be longer than span, since you're calling `New`

Reply:
Oh, I just noticed that I wrongly assumed that in the `New` the data also should be smaller than the span. Reading carefully your comments again I noticed that this is only the case for the `NewWithDataSpan`. I adjusted the code and the tests for both cases now.